_posts/2023-10-13-authz-keto-introduction.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title:  "Exploring Relation-based Models"
```

Comment:
a title is one of the hardest parts about a blog post. here is some feedback:

I think the title could be catchier, to make more people klick on it.

this story about the Google paper is fascinating. maybe the title can be a little more engaging using that story as a hook?

ideas 
- how google drive checks permission across millions of documents in 10ms
- learning from google: relation-based permissions scaled to millions of users
- permissons untangled: using relations instead of [old approach]

Reply:
Actually, this was the original title: From Google's Zanzibar to Ory Keto: Exploring the Evolution of Authorization Frameworks

But from my perspective that has too much focus on google zansibar, while it is quire interesting part the main point of the article is to make the integration clear and the zansibar part was just to explain the root. 

Maybe I should think about it.

Reply:
* Google's Zanzibar and Beyond: A Deep Dive into Relation-based Authorization
* Standing on the Shoulders of Giants: Relation-Based Authz Inspired by Google Zanzibar

Reply:
I think your last suggestions are nice